bugbug/models/qaneeded.py

```diff
@@ -56,6 +56,9 @@ def __init__(self, lemmatization=False):
         self.clf = xgboost.XGBClassifier(n_jobs=16)
         self.clf.set_params(predictor='cpu_predictor')
 
+    def rollback(self, change):
+        return change['field_name'] in ['qawanted', 'qe-verify']
```

Comment:
Analyze a few bugs manually, IIRC qe-verify is never in 'field_name'.

Reply:
Yes, neither is `qawanted`. `field_name` contains `qa-contact`, which is the closest it comes.

Reply:
Yep. So you'll need to use some other value from `change`.

Reply:
We need to rollback to when `qaneeded` or `qe-verify` was added, right? So maybe setting change['added'] in `['qawanted', 'qe-verify']` should work?

Reply:
Have you checked if they can be in change['added']?

Reply:
Yes, they are present in `change['added']`.

Reply:
Good, then it should work!
Please also add a test with a bug with qawanted and one with qe-verify.

Reply:
Also, if they are present in change['added'], then we should update get_labels to take that into account.

Reply:
`get_labels` already uses this information to give bugs a target label.

Reply:
Looking at the bug you used for the test, this doesn't work. The bug has added == qe-verify+.

Reply:
Fixed that. Added the `startswith` method for this.